src/sentry/integrations/discord/utils/auth.py

```diff
@@ -1,11 +1,6 @@
-from cryptography.exceptions import InvalidSignature
 from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PublicKey
 
 
-def verify_signature(public_key_string: str, signature: str, message: str) -> bool:
-    try:
-        public_key = Ed25519PublicKey.from_public_bytes(bytes.fromhex(public_key_string))
-        public_key.verify(bytes.fromhex(signature), message.encode("utf-8"))
-        return True
-    except (InvalidSignature, ValueError):
-        return False
+def verify_signature(public_key_string: str, signature: str, timestamp: str, body: str) -> None:
```

Comment:
I'd probably leave this as it was so you can validate the error through the logging added